components/auto-complete/auto-complete-dropdown.tsx

```diff
@@ -21,35 +24,61 @@ export type AutoCompleteDropdownProps = Props & typeof defaultProps & NativeAttr
 
 const AutoCompleteDropdown: React.FC<React.PropsWithChildren<AutoCompleteDropdownProps>> = ({
   children,
+  variant,
   visible,
   className,
   dropdownStyle,
   disableMatchWidth,
 }) => {
+  const isSolid = useMemo(() => variant === 'solid', [variant])
```

Comment:
I suggest not to use useMemo here, because equal condition is not expensive calculations, and the dependance array list will make equal check every time too.

Reply:
done